src/content/1.7/development/components/grid/tutorials/work-with-grid-actions.md

```diff
@@ -171,17 +171,13 @@ final class MySpecificAction extends AbstractBulkAction
 Then you need to create template so it can render nicely in your grid.
 
 ```twig
-{# mymodule/views/PrestaShop/Admin/Common/Grid/Actions/my_specific.html.twig #}
+{# mymodule/views/PrestaShop/Admin/Common/Grid/Actions/Grid/my_specific.html.twig #}
```

Comment:
Grid twice :thinking:

Reply:
Yes that's how the templates are stored in the code, there are three different folders the doc was just not up to date